src/ShearStressTransportEquationSystem.C

```diff
@@ -524,6 +539,7 @@ ShearStressTransportEquationSystem::post_iter_work()
 
     auto ngpIddesRans = fieldMgr.get_field<double>(
       get_field_ordinal(meta, "iddes_rans_indicator"));
+    ngpIddesRans.modify_on_device();
```

Comment:
@alanw0  yes it is here in the `post_iter_work` function This is a conditional evaluation that only occurs when the IDDES model is being used.